region/brinstar/blue.json

```diff
@@ -1219,10 +1219,40 @@
                   "obstacles": [
                     {
                       "id": "A",
-                      "requires": [ "h_canDestroyBombWalls" ]
+                      "requires": [
+                        {"or": [
+                          "h_canUsePowerBombs",
+                          "ScrewAttack"
+                        ]}
+                      ]
+                    }
+                  ],
+                  "note": "Base strat when entering the room from below.",
+                  "devNote": "canWallJumpIntoMorph is not needed here, as the precision is very low."
+                },
+                {
+                  "name": "Power Bomb, Wall Jump, and Bombs",
```

Comment:
Walljump isn't needed here either.  The key feature is placing a bomb right against the bomb block.